lib/also_energy/client.rb

```diff
@@ -2,15 +2,20 @@
 require 'pry'
 require 'virtus'
 require './lib/also_energy/hash_wrangler'
+require './lib/also_energy/connection'
 require './lib/also_energy/site'
 require './lib/also_energy/hardware'
 
 module AlsoEnergy
+
+  class << self; attr_accessor :session_id; end
```

Comment:
I don't think we can store this as a class variable. Different instances will have different session ids (think multiple customers using the app)